src/phylum/init/cli.py

```diff
@@ -38,33 +38,43 @@
 from phylum.github import github_request
 from phylum.init import SCRIPT_NAME
 from phylum.init.sig import verify_sig
-from phylum.logger import LOG, MARKUP_NO_HI, progress_spinner, set_logger_level
+from phylum.logger import LOG, progress_spinner, set_logger_level
 
 
-def get_phylum_settings_path():
+def get_phylum_settings_path() -> Path:
     """Get the Phylum settings path and return it."""
-    home_dir = pathlib.Path.home()
+    home_dir = Path.home()
 
-    config_home_path = os.getenv("XDG_CONFIG_HOME")
+    config_home_path = os.getenv("XDG_CONFIG_HOME", "")
     if not config_home_path:
-        config_home_path = home_dir / ".config"
+        config_home_path = str(home_dir / ".config")
 
-    phylum_config_path = pathlib.Path(config_home_path) / "phylum" / "settings.yaml"
+    phylum_config_path = Path(config_home_path) / "phylum" / "settings.yaml"
```

Comment:
Will this just be `C:\User\me\phylum\settings.yaml` on Windows?

Reply:
Close...it will be `C:\Users\me\.config\phylum\settings.yaml`. See the screenshots here for what that looks like in action.

Reply:
That's not a standard Windows path though, right? Do you just not care because this is CI?

I'd usually expect this in `%APPDATA%`.

Reply:
That path comes from the Phylum CLI. If it needs to change for Windows, it will need to be updated there. I don't think that is important at this time, since the primary use case for this feature is on a Windows server in CI (for a single customer that likely doesn't care...but we'll find out soon enough).

Reply:
> ...and dirs::config_dir() looks to XDG_CONFIG_HOME first and falls back to using the .config directory in the user's home directory otherwise.

This is not accurate on Windows:

https://docs.rs/dirs/latest/dirs/fn.config_dir.html

Reply:
Am I missing something? From what I can tell, the `dirs::config_dir()` function comes from a local crate - `cli/src/dirs.rs` - and not the one from crates.io

Reply:
Yeah that's my bad, I got confused there. That said, I'd say if we wanted to support Windows, we should be using the `dirs` crate.

Reply:
New issue created: https://github.com/phylum-dev/cli/issues/1509